reolink_aio/api.py

```diff
@@ -3222,6 +3222,194 @@ async def reboot(self, channel: int | None = None) -> None:
+                _LOGGER.warning(
+                    "CertificateClear: API returned code %s, response code %s/%s (may be expected if no certificates exist)",
+                    json_data[0]["code"],
+                    rspCode,
+                    detail,
+                )
+                # Don't raise error - camera may not have certificates to clear
+                return True
+
+            _LOGGER.info("Successfully cleared certificates from %s", self.nvr_name)
+            return True
+
+        except ReolinkError as err:
+            _LOGGER.error("Error clearing certificates from %s: %s", self.nvr_name, err)
+            raise
+
+    async def import_certificate(
+        self,
+        cert_content: str,
+        key_content: str,
+        cert_name: str = "server",
+    ) -> bool:
+        """
+        Import SSL/TLS certificate and private key to the device.
+
+        Args:
+            cert_content: PEM-encoded certificate content.
+            key_content: PEM-encoded private key content.
+            cert_name: Base name for certificate files (default: "server").
+
+        Returns:
+            True if import succeeded.
+
+        Raises:
+            InvalidParameterError: If certificate or key content is invalid.
+            ApiError: If the API returns an error.
+
+        Note:
+            - The device only supports RSA certificates, not EC certificates.
+            - Some models (e.g., E1 Pro) may require a reboot to activate the certificate.
+        """
+        if not cert_content or not key_content:
+            raise InvalidParameterError("Certificate and key content must not be empty")
+
+        # Encode certificate and key to base64
+        cert_b64 = base64.b64encode(cert_content.encode("utf-8")).decode("utf-8")
+        key_b64 = base64.b64encode(key_content.encode("utf-8")).decode("utf-8")
+
+        # Calculate sizes
+        cert_size = len(cert_content.encode("utf-8"))
+        key_size = len(key_content.encode("utf-8"))
+
+        body = [
+            {
+                "cmd": "ImportCertificate",
+                "action": 0,
+                "param": {
+                    "importCertificate": {
+                        "crt": {
+                            "size": cert_size,
+                            "name": f"{cert_name}.crt",
+                            "content": cert_b64,
+                        },
+                        "key": {
+                            "size": key_size,
+                            "name": f"{cert_name}.key",
+                            "content": key_b64,
+                        },
+                    }
+                },
+            }
+        ]
+
+        _LOGGER.info(
+            "Importing certificate to %s (cert: %s bytes, key: %s bytes)",
+            self.nvr_name,
+            cert_size,
+            key_size,
+        )
+
+        try:
+            json_data = await self.send(body, expected_response_type="json")
```

Comment:
```suggestion
            json_data = await self.send_setting(body)
```

Then you don't need the check for a ApiError